app/models/user.rb

```diff
@@ -7,4 +7,8 @@ class User < ApplicationRecord
   validates :first_name, presence: true
   enum role: { default: 0, admin: 1 }
   has_secure_password
+
+  def repositories
+    Github.list_repos(token)
```

Comment:
I am not sure if this should live here... we may want to use a method call to this being created elsewhere? It just doesn't feel like a User should know how to create a Github...

Reply:
So a Github isn't being created here, the user is just calling the Github class which has a method to list a user's repos based on their token.